airbyte-cdk/python/connector_builder/connector_builder_source.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from datetime import datetime
+from typing import Any, Mapping, Union
+
+from airbyte_cdk.models import AirbyteMessage, AirbyteRecordMessage
+from airbyte_cdk.sources.declarative.manifest_declarative_source import ManifestDeclarativeSource
+from airbyte_cdk.utils.traced_exception import AirbyteTracedException
+
+
+class ConnectorBuilderSource:
+    def __init__(self, source: ManifestDeclarativeSource):
+        self.source = source
+
+    def list_streams(self) -> AirbyteRecordMessage:
+        raise NotImplementedError
+
+    def stream_read(self, command_config) -> AirbyteRecordMessage:
+        raise NotImplementedError
+
+    @staticmethod
+    def _emitted_at():
+        return int(datetime.now().timestamp()) * 1000
+
+    def resolve_manifest(self) -> Union[AirbyteMessage, AirbyteRecordMessage]:
+        try:
+            return AirbyteRecordMessage(
+                data={"manifest": self.source.resolved_manifest},
+                emitted_at=self._emitted_at(),
+                stream="",
```

Comment:
would it make sense to emit the records in a `manifest` or `resolved_manifest` stream?

Reply:
Mind clarifying? I'm not sure I see what you're suggesting.

Reply:
I'm suggesting setting the stream name for these records

Reply:
Ah okay good call, updated.